tests/tests_twobody/test_orbit.py

```diff
@@ -1034,21 +1034,17 @@ def test_from_coord_if_coord_is_not_of_shape_zero():
 def test_from_sbdb_and_from_horizons_give_similar_results(target_name):
     ss_target = Orbit.from_sbdb(target_name)
     ss_classical = ss_target.classical()
-
-    ss_ref = Orbit.from_horizons(
-        name=target_name, attractor=Sun, plane=Planes.EARTH_ECLIPTIC
-    )
-
-    ss_ref = ss_ref.propagate_to_anomaly(
```

Comment:
Which was the aim of propagating the orbit in the old version? I mean, the method returns the orbit with that specific true anomaly, isn't it?

Reply:
What I understood, this test was to double check that the data from sddb and from horizon gave the same results. In order to do so, I send the same epoch in both methods and the result was positive.

Reply:
> What I understood, this test was to double check that the data from sddb and from horizon gave the same results. In order to do so, I send the same epoch in both methods and the result was positive.

Yes, that was the idea of #867!